include/seqan/basic/triple_packed.h

```diff
@@ -61,7 +61,7 @@ namespace seqan {
  * @tparam T3 The third type of the Triple.
  */
 
-#ifdef PLATFORM_WINDOWS
+#ifdef STD_LIB_VS
     #pragma pack(push,1)
```

Comment:
Test programs:

*  Test using `packed`
```c++
struct test
{
    unsigned char  field1;
    unsigned short field2;
    unsigned long  field3;
} __attribute__((__packed__));

typedef struct test test_t;


int main() {
    test_t var1, var2;
    return 0;
}
```
* Test using `pragma pack`
```c++
#pragma pack(push,1)
struct test
{
    unsigned char  field1;
    unsigned short field2;
    unsigned long  field3;
};
#pragma pack(pop)

typedef struct test test_t;

int main() {
    test_t var1, var2;
    return 0;
}
```

| os    | compiler | packed | pragma pack |
--------| ---------- | ---------- | ---------------- |
| linux | gcc | ✔  | ✔  |
| linux | clang | ✔  | ✔  |
| linux | icc | ✔  | ✔  |
| windows | msvc |  ✖   | ✔  |
| windows | clang/llvm |  ✔   | ✔  |
| windows | clang/c2 |  not building   | not building |
| windows | icc |  ✖   | ✔  |

All seem to support `#pragma pack`, [gcc](https://gcc.gnu.org/onlinedocs/gcc-5.1.0/gcc/Structure-Packing-Pragmas.html) also states this explicitly (they state that they give compatibility with msvc syntax). [Icc](https://software.intel.com/en-us/forums/intel-c-compiler/topic/297934#comment-1594636) states this vaguely. [clang](https://github.com/llvm-mirror/clang/blob/master/test/Parser/pragma-pack.c) has test case that indicates that they support it.

* Windows icc 
  ```
  cmake ../src -T"Intel C++ Compiler 16.0" -G"Visual Studio 14 2015 Win64"  -DCMAKE_CXX_FLAGS="/Qwd3373,2102 /D__is_assignable=__is_trivially_assignable"
  ```
* Windows clang/llvm
  ```
  cmake ../src -TLLVM-vs2014 -G"Visual Studio 14 2015 Win64"
  ```
* Windows clang/v140_clang_3_7 ([cmake](http://public.kitware.com/pipermail/cmake/2016-April/063185.html) added support in april in the nightlies, might be supported in the upcoming cmake3.6 release) 
  ```
  cmake ../src -TLLVM-vs2014 -G"Visual Studio 14 2015 Win64"
  ```